blog/config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  get 'welcome/index'
+ 
+  resources :articles
```

Comment:
I personally prefer to keep `root` at the top of the `Routes` file.

This way, when someone open the file, he directly knows where it starts.

Try to keep things organized as much as possible. ;)